### PR TITLE
Currency change feature

### PR DESCRIPTION
**Pull request by williecardoza**
-Added currency change functionality and stored in local storage
-Styled Nav, and bitcoin charts

### CODE REVIEW

File: src/components/NavBar/Dropdown/Dropdown.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+          >
+            <CurrencySymbol>€</CurrencySymbol>
+            <div>EUR</div>
+          </DropdownItem>
+          <DropdownItem
+            onClick={() => this.props.handleSelectedCurrency("GBP")}
+          >
+            <CurrencySymbol>£</CurrencySymbol>
+            <div>GBP</div>
+          </DropdownItem>
```
Comment (yoloOnTheBattlefield) [inline src/components/NavBar/Dropdown/Dropdown.js:30]: can you make this using an array?

File: src/components/NavBar/NavBar.js [hunk 2]
```diff
@@ -1,53 +1,108 @@
...
+                <Wrapper>
+                  <CurrencyContainer onClick={this.handleDropdown}>
+                    <CurrencySymbol>
+                      {this.state.currency === "USD"
+                        ? "$"
+                        : this.state.currency === "EUR"
+                        ? "€"
+                        : this.state.currency === "GBP"
+                        ? "£"
+                        : ""}
```
Comment (yoloOnTheBattlefield) [inline src/components/NavBar/NavBar.js:90]: maybe use one universal object where you can grab these values here and above where i commented

**Review by yoloOnTheBattlefield — CHANGES_REQUESTED** [review]

File: src/components/NavBar/Dropdown/Dropdown.js (hunk 1 above)
Comment (williecardoza) [reply src/components/NavBar/Dropdown/Dropdown.js]: fixed

File: src/components/NavBar/NavBar.js (hunk 2 above)
Comment (williecardoza) [reply src/components/NavBar/NavBar.js]: Placed currencies into an array and mapped through them

**Review by yoloOnTheBattlefield — APPROVED** [review]

**Pull request merged**